### DISCUSSION TITLE
Multiple items deposit

### DISCUSSION THREAD

**Discussion by Kpa6i** [voxeltycoon/issues#438] [Modding]
Can you add the support of multiple items to deposit?
That will be great things for modding.
```
"ItemURI": "item.item"
```
replace with
```
"Items: [
  "item1.item",
  "item2.item"
]
```

**Comment by victorleblais** [comment]
Good idea !

How would that work?

You would be able to place two (or more) types of mine on the same deposit?

Would a mine be able to extract both? If so, how would it work with conveyor belt outputs?

**Comment by Kpa6i** [comment]
@victorleblais In my conception, several different mines can handle one deposit  (for example some sort of minerals that are always together in nature), then the player places two different mines for each mineral.

When we will have a feature with two outputs from the device this feature can be a rewriting to allow a player to select what to extract from a deposit from one mine.
